lib/pages/new_space/new_space.dart

```diff
@@ -150,7 +150,10 @@ class NewSpaceController extends State<NewSpace> {
     try {
       final avatar = this.avatar;
       avatarUrl ??= avatar == null ? null : await client.uploadContent(avatar);
-
+      final classCode = await SpaceCodeUtil.generateSpaceCode(client);
```

Comment:
It would be good to add some sort of error here (to be caught upstream), or give some indication of an error here. Otherwise, the form just doesn't go through, which is confusing.

Reply:
Fix in latest commit

Reply:
Made class code not null, generate class code will throw and error that will be caught by overarching try except and display errors